src/session.rs

```diff
@@ -129,7 +129,7 @@ impl SessionStoreBroker {
             Err(_) => return false,
         };
         let result = unsafe { cb(userdata, &key, &value) };
-        result == rustls_result::Ok as u32
```

Comment:
note to self: this diff can be reverted, it was a side-effect of when I had the enum macro spitting out variants without their explicit value.

Reply:
Reverted.